api/api.go

```diff
@@ -50,8 +50,10 @@ func New(ctx context.Context, cfg *config.Config, r chi.Router, idc *identity.Cl
 
 func (api *API) enablePublicEndpoints() {
 	api.Router.Post("/filters", api.createFilter)
+	api.Router.Post("/filter-outputs", api.createFilterOutput)
```

Comment:
```suggestion
        api.Router.Post("/filter-outputs", api.createFilterOutput)
	api.Router.Post("/filters", api.createFilter)
```

Reply:
I would swap the order of these in order to keep the `/filter` resources together.
Improves readability